packages/server/postgres/extensions/pgvector/Makefile

```diff
@@ -0,0 +1,20 @@
+.PHONY: all clone build install
+
+# Default target
+all: build
+
+# Target for handling the pgvector repository
+clone:
+	if [ ! -d "pgvector" ]; then \
+		git clone https://github.com/ankane/pgvector.git; \
```

Comment:
we should install ankane's version on our GH test action otherwise any reference to vector will break

https://github.com/ParabolInc/parabol/blob/ec5d15bd851430cc0856a1455ee8c741ec40b26a/.github/workflows/build.yml#L14-L17

Reply:
kk, will change